startup.sh

```diff
@@ -2,36 +2,16 @@
 
 bundle check || bundle install
 
-if [[ $1 == "--integration" ]] ; then
-  GOVUK_APP_DOMAIN=www.integration.publishing.service.gov.uk \
-  GOVUK_WEBSITE_ROOT=https://www.integration.publishing.service.gov.uk \
-  PLEK_SERVICE_CONTENT_STORE_URI=${PLEK_SERVICE_CONTENT_STORE_URI-https://www.integration.publishing.service.gov.uk/api} \
-  PLEK_SERVICE_STATIC_URI=${PLEK_SERVICE_STATIC_URI-assets.integration.publishing.service.gov.uk} \
-  PLEK_SERVICE_SEARCH_URI=${PLEK_SERVICE_SEARCH_URI-https://www.integration.publishing.service.gov.uk/api} \
-  bundle exec rails s -p 3070
-
-elif [[ $1 == "--staging" ]] ; then
-  GOVUK_APP_DOMAIN=www.staging.publishing.service.gov.uk \
-  GOVUK_WEBSITE_ROOT=https://www.staging.publishing.service.gov.uk \
-  PLEK_SERVICE_CONTENT_STORE_URI=${PLEK_SERVICE_CONTENT_STORE_URI-https://www.staging.publishing.service.gov.uk/api} \
-  PLEK_SERVICE_STATIC_URI=${PLEK_SERVICE_STATIC_URI-assets.staging.publishing.service.gov.uk} \
-  PLEK_SERVICE_SEARCH_URI=${PLEK_SERVICE_SEARCH_URI-https://www.staging.publishing.service.gov.uk/api} \
-  bundle exec rails s -p 3070
-
-elif [[ $1 == "--live" ]] ; then
+if [[ $1 == "--live" ]] ; then
   GOVUK_APP_DOMAIN=www.gov.uk \
   GOVUK_WEBSITE_ROOT=https://www.gov.uk \
   PLEK_SERVICE_CONTENT_STORE_URI=${PLEK_SERVICE_CONTENT_STORE_URI-https://www.gov.uk/api} \
   PLEK_SERVICE_STATIC_URI=${PLEK_SERVICE_STATIC_URI-assets.publishing.service.gov.uk} \
   PLEK_SERVICE_SEARCH_URI=${PLEK_SERVICE_SEARCH_URI-https://www.gov.uk/api} \
   bundle exec rails s -p 3070
-elif [[ $1 == "--dummy" ]] ; then
-  GOVUK_APP_DOMAIN=www.gov.uk \
-  GOVUK_WEBSITE_ROOT=https://www.gov.uk \
-  PLEK_SERVICE_CONTENT_STORE_URI=${PLEK_SERVICE_CONTENT_STORE_URI-https://govuk-content-store-examples.herokuapp.com/api} \
-  PLEK_SERVICE_STATIC_URI=${PLEK_SERVICE_STATIC_URI-assets.publishing.service.gov.uk} \
-  PLEK_SERVICE_SEARCH_URI=${PLEK_SERVICE_SEARCH_URI-https://www.gov.uk/api} \
-  bundle exec rails s -p 3070
 else
-  bundle exec rails s -p 3070
+  echo "ERROR: other startup modes are not supported"
```

Comment:
Would it make sense to remove the if and just run it in --live mode regardless? Maybe echo out that it's pulling data from live as it starts, or at the end for clarity?

Reply:
I wondered about this too. A couple of reasons for keeping it as-is are:

- In theory everyone should only be typing `./startup.sh --live`, and this means that will continue working.
- I think it's helpful to have the concept of "live" here, as it mirrors the "app-live" stack in GOV.UK Docker.

I can also see the convenience of not having to typing the flag. On balance, I don't think there's a particularly strong argument for either option, so I'm going to leave it as-is.